api/internal/handler/data_loader/route_export.go

```diff
@@ -245,8 +246,14 @@ func (h *Handler) RouteToOpenAPI3(c droplet.Context, routes []*entity.Route) (*o
 		path.RequestBody = &openapi3.RequestBodyRef{Value: requestBody}
 		path.Responses = openapi3.NewResponses()
 
-		for i := range route.Methods {
-			switch strings.ToUpper(route.Methods[i]) {
+		if route.Methods != nil && len(route.Methods) > 0 {
+			routeMethods = route.Methods
+		} else {
+			routeMethods = []string{http.MethodGet, http.MethodPost, http.MethodPut, http.MethodDelete, http.MethodPatch, http.MethodHead, http.MethodConnect, http.MethodTrace, http.MethodOptions}
```

Comment:
Can we use a pre-defined unexportable variable to indicate this default method set?

Reply:
Yeah,Because these methods such as "GET", "POST" and so on are immutable, the use here is more uniform, and "Get", "get" and "GET" will not appear.

Reply:
I mean creating a variable `_allHTTPMethods`, which is:

```go
var (
    _allHTTPMethods = []string{http.MethodGet, http.MethodPost, ...}
)
```

Reply:
Sorry, I misunderstood.
Done. Thks